coretex/configuration.py

```diff
@@ -62,6 +63,7 @@ def loadConfig() -> Dict[str, Any]:
         try:
             config: Dict[str, Any] = json.load(configFile)
         except json.JSONDecodeError:
+            logging.getLogger("cli").debug(f"Failed to load corrupted config file. Content: {configFile.read()}")
```

Comment:
Log out the exception as well with a full stack trace. Also `json.load` will read this file as well which will cause `configFile.read` to raise an error the 2nd time it is called. The file should only be read once

Reply:
So I can just use configFile as if I am calling configFile.read() after initial .read() is called?